### PR TITLE
Add recipe for RESTinstance

### PR DESCRIPTION
**Pull request by abloemert**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/restinstance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c241e44e51c7a793c7c5ca0476caac2b703cc78da8041ab05ded965ce49bc187
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
```
Comment (scopatz) [inline recipes/restinstance/meta.yaml:14]: This should be `noarch: python`

Comment (abloemert) [reply recipes/restinstance/meta.yaml:14]: I tried that, but than [the win-64 build fails](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=182256&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=26d9c833-f81c-502c-87bc-0cb0be44f838)

Comment (scopatz) [reply recipes/restinstance/meta.yaml:14]: That is OK.  Noarch: python only needs to build on linux.  It will run on any platform though

Comment (abloemert) [reply recipes/restinstance/meta.yaml:14]:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**